### PR TITLE
add my coordinates

### PR DESCRIPTION
**Pull request by devesh3710**
12.9716° N, 77.5946° E

### CODE REVIEW

**Comment by devesh3710** [comment]
12.9716° N, 77.5946° E

File: _pins/shobhit1.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: "shobhit1"
```
Comment (crichID) [inline _pins/shobhit1.json:2]: @devesh3710 great work on adding your pin - one small change, you will need to remove the quote marks `"` from your pin. After you have done that, I will be able to merge your pull request!

Comment (devesh3710) [reply _pins/shobhit1.json:2]: @Shobhit1 I guess this pertains to your pull request.

**Comment by devesh3710** [comment]
Closing this, seems I created some mistake.

**Pull request closed without merging**